integration/localnet/bootstrap.go

```diff
@@ -508,18 +459,37 @@ func prepareObserverService(i int, observerName string, agPublicKey string) Serv
 	// Observers have a unique naming scheme omitting node id being on the public network
 	dataDir, profilerDir := prepareServiceDirs(observerName, "")
 
-	observerService := Service{
-		Image: fmt.Sprintf("localnet-%s", DefaultObserverName),
+	observerService := defaultService(DefaultObserverName, dataDir, profilerDir, i)
+	observerService.Command = append(observerService.Command, []string{
```

Comment:
same here.  just get rid of `[]string{}...` altogether and directly append to command